frappe/core/doctype/file/file.py

```diff
@@ -87,7 +86,6 @@ def validate(self):
 			self.handle_is_private_changed()
 
 		if not self.is_folder:
-			self.validate_file_path()
```

Comment:
removed because `get_full_path` is already called in `validate_file_on_disk` below.